src/resourceNameToUrn.spec.ts

```diff
@@ -1,7 +1,9 @@
 import { resourceNameToUrn } from './resourceNameToUrn.js'
+import { describe, it } from 'node:test'
+import assert from 'node:assert/strict'
 
-describe('resourceNameToUrn()', () => {
-	it.each([
+void describe('resourceNameToUrn', () => {
```

Comment:
```suggestion
void describe('resourceNameToUrn()', () => {
```

We always use function name in describe.

Reply:
https://github.com/NordicSemiconductor/asset-tracker-cloud-coiote-aws-converter-js/pull/85/commits/5d82ae548dac757c132f49a0441b955f1995f46c